packages/medusa-core-utils/src/create-require-from-path.ts

```diff
@@ -1,7 +1,7 @@
-import Module from "module"
+const Module = require("module")
 import path from "path"
 
-const fallback = filename => {
+const fallback = (filename: string) => {
   const mod = new Module(filename)
 
   mod.filename = filename
```

Comment:
question: is mixing `require` and `import` on purpose; if so why? if not please fix :)